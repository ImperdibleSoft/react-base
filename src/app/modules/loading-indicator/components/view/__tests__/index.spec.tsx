import * as React from 'react';
import { shallow } from 'enzyme';

import View from '..';

describe('Loading indicator', () => {
  it('should render a valid loading online component', () => {
    const component = shallow(<View fixedSidebar={false} isLoading />);

    expect(component).toMatchSnapshot();
  });

  it('should render a valid online component', () => {
    const component = shallow(<View fixedSidebar={false} />);

    expect(component).toMatchSnapshot();
  });
});
