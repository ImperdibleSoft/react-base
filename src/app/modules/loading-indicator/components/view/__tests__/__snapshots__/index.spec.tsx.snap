// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading indicator should render a valid loading online component 1`] = `
<Fragment>
  <Wrapper
    fixedSidebar={false}
    isLoading={true}
  >
    <SubLine
      effect="increase"
      isLoading={true}
    />
    <SubLine
      effect="decrease"
      isLoading={true}
    />
  </Wrapper>
  <LineSpacer
    isLoading={true}
  />
</Fragment>
`;

exports[`Loading indicator should render a valid online component 1`] = `
<Fragment>
  <Wrapper
    fixedSidebar={false}
    isLoading={false}
  >
    <SubLine
      effect="increase"
      isLoading={false}
    />
    <SubLine
      effect="decrease"
      isLoading={false}
    />
  </Wrapper>
  <LineSpacer
    isLoading={false}
  />
</Fragment>
`;
